test/development/basic/next-rs-api.test.ts

```diff
@@ -641,7 +641,7 @@ describe('next.rs api', () => {
 
     const count = process.env.CI ? 300 : 1000
     for (let i = 0; i < count; i++) {
-      await next.patchFileFast(file, nextContent)
+      await next.patchFile(file, nextContent)
```

Comment:
Since we've made `patchFile` configurable at test module level, we don't need `patchFileFast`